tests/framework/utils.py

```diff
@@ -89,6 +89,22 @@ def get_files_from(find_path: str, pattern: str, exclude_names: list = None,
     return found
 
 
+def get_free_mem_ssh(ssh_connection):
+    """
+    Get how much free memory a guest sees, over an ssh connection.
```

Comment:
I would add that we return the amount of free memory in kB

Reply:
Done.